src/react-apps/applications/service-development/Dockerfile

```diff
@@ -37,7 +37,7 @@ COPY --from=generate-service-development /applications/service-development/dist/
 COPY --from=generate-service-development /applications/service-development/js/react/editor.worker.js .
 COPY --from=generate-service-development /applications/service-development/js/react/typescript.worker.js .
 COPY --from=generate-service-development /applications/service-development/dist/service-development.css .
-COPY --from=generate-service-development /applications/ux-editor/dist/react-app.js .
-COPY --from=generate-service-development /applications/ux-editor/dist/react-app.css .
+COPY --from=generate-service-development /applications/ux-editor/dist/ui-editor.js .
```

Comment:
Are these files still in use? (ui-editor.js + ui-editor.css, nextline)

Reply:
No I don't think so, removed them 🌍